### DISCUSSION TITLE
Opening and closing square brackets specially designed for pipe organ notation

### DISCUSSION THREAD

**Discussion by prokrusztesz** [musescore/MuseScore#24403] [Ideas]
The need for a square bracket that can be scaled and possibly have a function attached to it has been raised several times before. (https://musescore.org/en/node/300462) As an interim solution, what appears to be a square bracket from the Arpeggio palette could be used as a starting bracket, but a closing bracket would also be needed.
It is important that the bracket is freely scalable and able to span all bars in the system. In the notation of the pipe organ, the range of the register and the manual to be used are usually indicated by such a bracket. See: https://musescore.org/sites/musescore.org/files/2021-08/IMG_20210805_163843.jpg

**Comment by cbjeukendrup** [comment]
Is this related to https://github.com/musescore/MuseScore/issues/24404?

**Reply by prokrusztesz** [reply]
Absolutely yes.

**Reply by cbjeukendrup** [reply]
Since that one is an "issue" and can therefore be added to our backlog, as opposed to this "discussion", let's close this discussion then, and follow that issue instead. Thanks!
